modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-configure-ingress-access-logging_{context}"]
+= Configure Ingress access logging
+
+You can configure the Ingress Controller to enable access logs for the Ingress Controller. If you have clusters that do not receive much traffic, then you can log to a sidecar. If you have high traffic clusters, to avoid exceeding the capacity of the logging stack or integrate with a logging infrastructure outside of {product-title}, you can forward logs to a custom syslog endpoint. You can also specify the format for access logs.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+Configure Ingress access logging to a sidecar.
+
+* To configure Ingress access logging, you must specify a destination using `spec.logging.access.destination`. To specify logging to a sidecar destination, you must specify `container` `spec.logging.access.destination.type`. The following example is the definition of an Ingress Controller that logs to a `container` destination:
++
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  replicas: 2
+  endpointPublishingStrategy:
+    type: NodePortService
```

Comment:
From the provided example, it is worth mentioning that `NodePortService`  is not a necessary requirement for the feature to function. The logging options works with any endpoint strategy.